.drone.star

```diff
@@ -2233,3 +2248,137 @@ def buildAndTestDesignSystem(ctx):
+        },
+        {
+            "name": "keycloak",
+            "image": KEYCLOAK,
+            "detach": True,
+            "environment": {
+                "OCIS_DOMAIN": "ocis:9200",
+                "KC_HOSTNAME": "keycloak:8443",
+                "KC_DB": "postgres",
+                "KC_DB_URL": "jdbc:postgresql://postgres:5432/keycloak",
+                "KC_DB_USERNAME": "keycloak",
+                "KC_DB_PASSWORD": "keycloak",
+                "KC_FEATURES": "impersonation",
+                "KEYCLOAK_ADMIN": "admin",
+                "KEYCLOAK_ADMIN_PASSWORD": "admin",
+                "KC_HTTPS_CERTIFICATE_FILE": "./keycloak-certs/keycloakcrt.pem",
+                "KC_HTTPS_CERTIFICATE_KEY_FILE": "./keycloak-certs/keycloakkey.pem",
+            },
+            "commands": [
+                "mkdir -p /opt/keycloak/data/import",
+                "cp tests/drone/ocis_keycloak/ocis-ci-realm.dist.json /opt/keycloak/data/import/ocis-realm.json",
+                "/opt/keycloak/bin/kc.sh start-dev --proxy edge --spi-connections-http-client-default-disable-trust-manager=false --import-realm --health-enabled=true",
+            ],
+            "volumes": [
+                {
+                    "name": "certs",
+                    "path": "/keycloak-certs",
+                },
+            ],
+        },
+        {
+            "name": "wait-for-keycloak",
+            "image": OC_CI_WAIT_FOR,
+            "commands": [
+                "wait-for -it keycloak:8443 -t 300",
+            ],
+        },
+    ]
+
+def e2eTestsOnKeycloak(ctx):
+    if not "full-ci" in ctx.build.title.lower() and ctx.build.event != "cron":
+        return []
+
+    return [{
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "e2e-test-on-keycloak",
+        "workspace": web_workspace,
+        "steps": restoreBuildArtifactCache(ctx, "pnpm", ".pnpm-store") +
+                 restoreBuildArtifactCache(ctx, "playwright", ".playwright") +
+                 installPnpm() +
+                 installPlaywright() +
+                 keycloakService() +
+                 restoreBuildArtifactCache(ctx, "web-dist", "dist") +
+                 setupServerConfigureWeb("2") +
+                 restoreOcisCache() +
+                 ocisService("keycloak") +
+                 [
+                     {
+                         "name": "e2e-tests",
+                         "image": OC_CI_NODEJS,
+                         "environment": {
+                             "BASE_URL_OCIS": "ocis:9200",
+                             "HEADLESS": "true",
+                             "RETRY": "1",
+                             "REPORT_TRACING": "true",
+                             "KEYCLOAK": "true",
+                             "KEYCLOAK_HOST": "https://keycloak:8443",
+                         },
+                         "commands": [
+                             "pnpm test:e2e:cucumber tests/e2e/cucumber/features/journeys/kindergarten.feature",
```

Comment:
We are currently running `/journeys/kindergarten.feature` feature only that covers login/upload/download/sharing. We wanted to run minimal tests with keycloak. @ScharfViktor what other scenarios would you recommend to run?
```suggestion
                             "pnpm test:e2e:cucumber tests/e2e/cucumber/features/journeys/kindergarten.feature",
```